app/models/relationship.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class Relationship < ApplicationRecord
+  validates :follower_id, presence: true
```

Comment:
You should also validate that an user can only follow the same user once, and viceversa.

You can add this logic with `uniqueness: { scope: ... }`